File: go-sdk/pkg/models/operations/putapidisplayadsidunpublish.go
package operations

type PutAPIDisplayAdsIDUnpublishPathParams struct {
	ID int32 `pathParam:"style=simple,explode=false,name=id"`
}

type PutAPIDisplayAdsIDUnpublishRequest struct {
	PathParams PutAPIDisplayAdsIDUnpublishPathParams
}

type PutAPIDisplayAdsIDUnpublishResponse struct {
	ContentType string
	StatusCode  int64
}

File: ts-sdk/src/sdk/models/operations/putapidisplayadsidunpublish.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class PutApiDisplayAdsIdUnpublishPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=id" })
  id: number;
}


export class PutApiDisplayAdsIdUnpublishRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: PutApiDisplayAdsIdUnpublishPathParams;
}


export class PutApiDisplayAdsIdUnpublishResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;
}
